common-libs/common/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 1.9.1 (2020-06-18)
+* Improved type declarations
+* Breaking changes
```

Comment:
As I understand, those are "internals" that were exposed? It's just doesn't sound right to update patch version with "breaking changes" in changelog :D